naga/src/span.rs

```diff
@@ -11,6 +11,7 @@ pub struct Span {
 
 impl Span {
     pub const UNDEFINED: Self = Self { start: 0, end: 0 };
+
```

Comment:
This change here is only tangentially related to the subject matter of this PR, but I figured it didn't hurt to include.